shared/settings/notifications/render.tsx

```diff
@@ -49,56 +56,89 @@ const Group = (props: {
             subscribed={s.subscribed}
           />
         ))}
-    </Box>
+    </Kb.Box>
     {props.unsub && (
-      <Box style={{...globalStyles.flexBoxColumn}}>
-        <Text type="BodyBig">Or:</Text>
-        <Checkbox
-          style={{marginTop: globalMargins.small}}
+      <Kb.Box style={{...Styles.globalStyles.flexBoxColumn}}>
+        <Kb.Text type="BodySmall">Or</Kb.Text>
+        <Kb.Checkbox
+          style={{marginTop: Styles.globalMargins.xtiny}}
           onCheck={props.onToggleUnsubscribeAll || null}
           disabled={!props.allowEdit}
           checked={!!props.unsubscribedFromAll}
           label={`Unsubscribe me from all ${props.unsub} notifications`}
         />
-      </Box>
+      </Kb.Box>
     )}
-  </Box>
+    <Kb.Divider style={styles.divider} />
+  </Kb.Box2>
+)
+const EmailSection = (props: Props) => (
+  <Group
+    allowEdit={props.allowEdit}
+    groupName="email"
+    label="All email notifications will be sent to your primary email address."
+    onToggle={props.onToggle}
+    onToggleUnsubscribeAll={() => props.onToggleUnsubscribeAll('email')}
+    title="Email notifications"
+    unsub="mail"
+    settings={props.groups.email && props.groups.email.settings}
+    unsubscribedFromAll={props.groups.email && props.groups.email.unsubscribedFromAll}
```

Comment:
Does this need a few `!!`'s?

Reply:
I don't think so. It passes TS and the component just checks if settings is falsey.

Reply:
oh oops you meant unsub. That should be fine too, the component includes a `|| null`.

Reply:
Eh, I'd rather move it to where these are. You're right.